persistence/remote/src/test/java/org/infinispan/persistence/remote/RemoteStoreTest.java

```diff
@@ -146,8 +146,9 @@ protected PersistenceMarshaller getMarshaller() {
    }
 
    @Override
-   @AfterMethod
+   @AfterMethod(alwaysRun = true)
    public void tearDown() {
+      configuration = null;
```

Comment:
Why are the changes required for the `persistence/remote` tests?

Reply:
I changed the base store to not create a config every time, this was required for some of the database tests as each time we created a new config it would create a unique connection url. By setting the configuration to null ensures this test creates  a new config after each test, which is required for it to pass.